project/Dependencies.scala

```diff
@@ -36,7 +37,11 @@ object Dependencies {
 
   val zio                   = "dev.zio" %% "zio"                 % ZioVersion
   val `zio-cli`             = "dev.zio" %% "zio-cli"             % ZioCliVersion
+  val `zio-config`          = "dev.zio" %% "zio-config"          % ZioConfigVersion
+  val `zio-config-magnolia` = "dev.zio" %% "zio-config-magnolia" % ZioConfigVersion
+  val `zio-config-typesafe` = "dev.zio" %% "zio-config-typesafe" % ZioConfigVersion
   val `zio-json-yaml`       = "dev.zio" %% "zio-json-yaml"       % ZioJsonVersion
+  val `zio-parser`          = "dev.zio" %% "zio-parser"          % ZioParserVersion
```

Comment:
zio-parser implementation wasn't good enough (parser was very slow - about a minute runtime per call. equivalent scala-parser-combinators implementation was fast).
I ended up replacing with a good enough regex (with the help of co-pilot).
This dependency is currently not used, and can be removed.

Reply:
Please remove it then. Btw, we have RichTextCodec, which might help.